src/components/GameView/CannotCounterDialog.vue

```diff
@@ -1,47 +1,62 @@
 <template>
-  <v-dialog v-model="show" persistent max-width="650">
-    <v-card v-if="oneOff" id="cannot-counter-dialog">
-      <v-card-title>Cannot Counter</v-card-title>
-      <v-card-text>
-        <span v-if="!opponentLastTwo">
-          Your opponent has played the {{ oneOff.name }} as a one-off
-          <span v-if="target"> targeting your {{ target.name }}</span>
+  <base-dialog v-if="oneOff" v-model="show" title="Cannot Counter" id="cannot-counter-dialog">
+    <template #body>
+      <div v-if="!opponentLastTwo" class="my-2">
+        Your opponent has played the 
+        <game-card-name :card-name="oneOff.name" />
```

Comment:
This new component basically serves to make the card names easier to read against the dark chocolate background (surface-1) by wrapping them in the lighter cream color (surface-1) with some padding 
![image](https://user-images.githubusercontent.com/6520704/233659543-8b287f3d-09f4-432f-8cd0-92166249dd89.png)


Interestingly it adds a bit of a gotcha when testing because the assertions for text content need to account for nested elements with the `{includeShadowDom: true}` option